db/schema_tables.cc

```diff
@@ -156,7 +156,7 @@ static future<user_types_to_drop> merge_types(distributed<service::storage_proxy
 static future<> merge_functions(distributed<service::storage_proxy>& proxy, schema_result before, schema_result after);
 static future<> merge_aggregates(distributed<service::storage_proxy>& proxy, schema_result before, schema_result after);
 
-static future<> do_merge_schema(distributed<service::storage_proxy>&, std::vector<mutation>, bool do_flush);
+static future<> do_merge_schema(distributed<service::storage_proxy>&, std::vector<mutation>, bool bypass_commitlog);
```

Comment:
Doesn't that invert the sense? previously, true == safe. Now, true == unsafe.

Safer to keep the sense in case a caller is missed.

Reply:
True, will change